pages/common/git-switch.md

```diff
@@ -12,14 +12,14 @@
 
 `git switch --create {{branch_name}}`
 
-- Create a new branch based on an existing commit:
+- Create a new branch based on an existing commit and switch to it:
 
 `git switch --create {{branch_name}} {{commit}}`
 
-- Update all submodules to match the target branch:
+- Switch to a branch and update all submodules to match the target branch:
```

Comment:
Maybe "...update all submodules to match it"? Or "...update all submodules to the versions it specifies"?

Reply:
changed.